src/pages/ProfilePage.tsx

```diff
@@ -89,15 +101,17 @@ export default function ProfilePage() {
         }
 
         const newProfileData: ProfileData = {
-          username: 'testuser',
+          username: data.username,
           email: data.email,
           name: data.name,
-          school: 'University of Maryland',
+          school: data.school,
           numAttempts: data.problemsAttempted,
           pieChartData: newPieChartData,
         }
 
+        setOldProfileData(newProfileData)
         setProfileData(newProfileData)
+        setIsLoadingPage(false)
       })
       .catch((error: Error) => {
         console.log(
```

Comment:
Looks like an old change, but is it safe to display this data to console?

Reply:
Definitely not. Good catch.